pyperf/pyperf_run

```diff
@@ -4,6 +4,7 @@ PATH="${PATH}:/usr/local/bin"
 export PATH
 python_pkgs=""
 python_exec=""
+PYPERF_VERSION="1.0.4"
```

Comment:
Considering future python versions don't work well with this pyperf version, I think we can bump it up to the current latest `1.11.0`.

@frival feel free to object :)